x-pack/plugins/infra/server/lib/domains/log_entries_domain/log_entries_domain.ts

```diff
@@ -135,9 +135,14 @@ export class InfraLogEntriesDomain {
       { field: '_index', value: document._index },
       { field: '_id', value: document._id },
     ];
+    const documentTimestamp = document._source[sourceConfiguration.fields.timestamp];
```

Comment:
For consistency, I would recommend sorting by the timestamp field too and using the value as you did for the `tiebreaker`. The `_source` contains the original timestamp format which could of a wide variety. The sort value on the other hand has already been normalized by Elasticsearch.

Then returning an `InfraTimeKey` as mentioned above should be simple.